tests.js

```diff
@@ -5005,29 +5005,8 @@ window.Specs = {
 					"dev": "#text-markers"
 				},
 				"tests": [
-					"disclosure-closed", "disclosure-open",
-					"hebrew",
-					"cjk-decimal", "cjk-ideographic",
-					"hiragana", "katakana", "hiragana-iroha", "katakana-iroha",
-					"japanese-informal", "japanese-formal", "korean-hangul-formal",
-					"korean-hanja-informal", "korean-hanja-formal",
-					"simp-chinese-informal", "simp-chinese-formal",
-					"trad-chinese-informal", "trad-chinese-formal",
-					"cjk-heavenly-stem", "cjk-earthly-branch",
-					"trad-chinese-informal", "trad-chinese-formal",
-					"simp-chinese-informal", "simp-chinese-formal",
-					"japanese-informal", "japanese-formal",
-					"arabic-indic", "persian", "urdu",
-					"devanagari", "gurmukhi", "gujarati",
-					"oriya", "kannada", "malayalam", "bengali",
-					"tamil", "telugu", "thai", "lao",
-					"myanmar", "khmer",
-					"hangul", "hangul-consonant",
-					"ethiopic-halehame", "ethiopic-numeric",
-					"ethiopic-halehame-am",
-					"ethiopic-halehame-ti-er", "ethiopic-halehame-ti-et",
-					"other-style", "inside", "outside", "\\32 style",
-					'"-"', "'-'",
+					'"-"',
+					"'-'",
 					"symbols(\"*\" \"\\2020\" \"\\2021\" \"\\A7\")",
```

Comment:
The `symbols()` syntax is actually also part of CSS Counter Styles Level 3, so it should be moved there. And it should be extended to cover [all the different values defined in the specification](https://drafts.csswg.org/css-counter-styles-3/#funcdef-symbols).

Reply:
Ok. I fix it.